src/EmployeesAdministrationApp.java

```diff
@@ -133,15 +136,43 @@ private static int chooseAnOptionFromMainMenu() {
 		options.put(6, ". Convert the type of an employee");
 		options.put(7, ". Exit Application");
 
-		System.out.println("Hello! Please select an option:\n" + "1. Add a new employee\n" + "2. View all employees\n"
+		System.out.println("\nPlease select an option:\n" + "1. Add a new employee\n" + "2. View all employees\n"
 				+ "3. Search an employee\n" + "4. Delete an employee\n" + "5. Update existing employee\n"
 				+ "6. Convert the type of an employee\n" + "7. Exit Application");
 		Scanner scan = new Scanner(System.in);
-		int number = scan.nextInt();
-		System.out.println("You have selected " + number + options.get(number));
-		return number;
+		int numberSelected = scan.nextInt();
+
+		System.out.println("You have selected " + numberSelected + options.get(numberSelected));
+		return numberSelected;
+	}
+
+	private static void searchEmployee() {
+		System.out.println("Please type in the id of the employee you want to search: ");
```

Comment:
this way of searching comes with a file reading; in other words, everytime you want to search something, you need to read the same content repetitively from a file. a more elegant solution would be to read only once from that file.